client/packages/common/src/utils/item/ItemUtils.ts

```diff
@@ -0,0 +1,23 @@
+import { useUrlQuery } from '@common/hooks';
+import { ItemNode } from '@common/types';
+import { RegexUtils } from '../regex';
+
+export const ItemUtils = {
```

Comment:
Hmm -- since this uses a hook (useUrlQuery), then that makes this a React hook too, so should we keep the naming convention: `useItemUtils` ? 

Technically, the inner functions are hooks (well, the "itemFilter" is, not the "matchItem"), not the ItemUtils object. But rather than have an object with several hooks inside it, I think it's nicer to just have the one hook `useItemUtils` which returns multiple methods (itemFilter, matchItem). (See `useIntlUtils` for an example of what I'm talking about).

Reply:
[e5ad277](https://github.com/openmsupply/open-msupply/pull/1930/commits/e5ad277e7900540582c55e13625d1b28c666ccda)